docs/src/api/plotting.md

```diff
@@ -0,0 +1,18 @@
+# Plotting
+
+JuliaDB has all access to all the power and flexibility of [Plots](https://github.com/JuliaPlots/Plots.jl)
+via [StatPlots](https://github.com/JuliaPlots/StatPlots.jl) and the `@df` macro.
+
+```
+using JuliaDB, StatPlots
+
+t = table(@NT(x = randn(100), y = randn(100)))
+
+@df t scatter(:x, :y)
+```
```

Comment:
Would be nice to make these actual examples with output... I guess the Plots and StatsPlots packages can be installed using Pkg.add on travis?

Reply:
Agreed.  I made some changes so that the plots are created by Documenter, but adds a bit to build times since now StatPlots + dependencies also get installed.  An alternative would be to link to a Jupyter notebook somewhere.